File: MyApp/SupabaseConfig.swift
import Foundation

/// Supabase project credentials.
///
/// Replace the placeholder anon (publishable) key with your project's key,
/// found in the Supabase dashboard under Settings → API.
enum SupabaseConfig {
    static let urlString = "https://cckklnxteyumsgiptfck.supabase.co"
    static let anonKey = "sb_publishable_FRfa8QyxuwnZ8dr167r39w_3OeXwq_a"

    /// True once the placeholders above have been replaced with real values.
    static var isConfigured: Bool {
        !urlString.contains("YOUR-PROJECT-REF") && anonKey != "YOUR-ANON-KEY"
    }

    static var url: URL {
        URL(string: urlString)!
    }
}

File: android/app/src/main/java/com/cbtipul/app/data/SupabaseConfig.kt
package com.cbtipul.app.data

object SupabaseConfig {
    const val URL = "https://cckklnxteyumsgiptfck.supabase.co"
    const val ANON_KEY = "sb_publishable_FRfa8QyxuwnZ8dr167r39w_3OeXwq_a"

    const val AUTH_CALLBACK = "cbtipul://auth-callback"
    const val PASSWORD_RESET_CALLBACK = "cbtipul://password-reset"

    val isConfigured: Boolean
        get() = !URL.contains("YOUR-PROJECT-REF") && ANON_KEY != "YOUR-ANON-KEY"
}
